impexp-core/src/main/java/org/citydb/core/database/adapter/AbstractSQLAdapter.java

```diff
@@ -153,6 +153,30 @@ public PreparedStatement prepareStatement(SQLStatement statement, Connection con
 		
 		return preparedStatement;
 	}
+
+	public List<String> getPlaceHolderValues(SQLStatement statement) throws SQLException {
+		List<PlaceHolder<?>> placeHolders = statement.getInvolvedPlaceHolders();
+
+		List<String> values = new ArrayList<>();
+		for (int i = 0; i < placeHolders.size(); i++) {
+			Object value = placeHolders.get(i).getValue();
+
+			String result = null;
+			if (value instanceof String || value instanceof Date || value instanceof Timestamp) {
+				result = "'" + value + "'";
+			} else if (value instanceof Boolean) {
+				result = ((Boolean) value) ? "1" : "0";
+			} else if (value instanceof Double || value instanceof Integer || value instanceof Long) {
```

Comment:
This `else if` should rather be changed into a general `else` to catch all possible cases.

Reply:
fixed with d948240